recipe/recipe_13TeV.sh

```diff
@@ -12,79 +13,48 @@ MINOR_VERSION=`echo $CMSSW_VERSION | sed "s|CMSSW_\([0-9]\)_\([0-9]\)_.*|\2|"`
 
 pushd $CMSSW_BASE/src
 
-echo "Checking out PAT dataformats"
-
-if [ "$LIMITS" = "1" ]
-then
-   echo ""     
-   echo "======================================="
-   echo "You shouldnt run limits/fits from here." 
-   echo "LIMITS should be run from 6XX, and do not require the rest of the machinery. Please, change area."
-   echo "Check https://twiki.cern.ch/twiki/bin/view/CMS/SWGuideHiggsAnalysisCombinedLimit for updates"
-   echo "======================================="
-   echo ""     
-fi
-
-
-if [ "$PATPROD" = "1" ]
-then
-  echo "Checking out tuple production tags"
-
-  echo "Need to update recipe for Quark Gluon Jet ID - which is the correct tag?"
-  #echo "Downloading Quark Gluon Jet ID"
-  #cvs co -r v1-2-3 -d QuarkGluonTagger/EightTeV UserCode/tomc/QuarkGluonTagger/EightTeV
-  # Quark-gluon tagging
-  git clone https://github.com/amarini/QuarkGluonTagger.git
-  pushd $CMSSW_BASE/src/QuarkGluonTagger
-  git checkout v1-2-6
-  pushd $CMSSW_BASE/src
+echo "Need to update recipe for Quark Gluon Jet ID - which is the correct tag?"
+#echo "Downloading Quark Gluon Jet ID"
+# Quark-gluon tagging
+git clone https://github.com/amarini/QuarkGluonTagger.git
```

Comment:
I think something fast and useful is to move all the "git clones" to one of the files the recipe.sh calls, and leave all the git cms-addpkg in another. What gives most problems is the git clone (since the directory is already there), but what will be changing more often is the addpkg or merge topics, which can be run on top of the running setup without problems. Then you can add a flag so that user decides what to call, the full thing, or only the git-cms things.